src/main/java/org/owasp/esapi/logging/slf4j/Slf4JLogFactory.java

```diff
@@ -56,7 +56,12 @@
     static {
         boolean encodeLog = ESAPI.securityConfiguration().getBooleanProp(DefaultSecurityConfiguration.LOG_ENCODING_REQUIRED);
         SLF4J_LOG_SCRUBBER = createLogScrubber(encodeLog);
-        SLF4J_LOG_APPENDER = createLogAppender();
+        
+    	boolean logClientInfo = true;
```

Comment:
@kwwall  @xeno6696  this should be the only line that needs to be updated in order to support the suppression of user content in the default log prefix.

Not sure why the indent is wonky, it looks correct in the IDE & file.